MetaMask/metamask-extension
# Remove openPopup argument from the preferences controller

## Explanation
Removed `openPopup` argument from the preferences controller, it's not being used in the controller.

* Fixes [ #399](https://github.com/MetaMask/MetaMask-planning/issues/399)

## Manual Testing Steps

- Go to settings
- Change any preference (show Test Networks)
- Restart the extension and check if changes are persisted

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
